kustomize/README.md

```diff
@@ -0,0 +1,228 @@
+  sed -i "" "s/<TERRASCAN_NAMESPACE>/terrascan/g" webhook/kustomization.yaml
+  sed -i "" "s/<TERRASCAN_NAMESPACE>/terrascan/g" webhook/validating-webhook.yaml
+  ```
+4. Ensure that your desired namespace exist.
+
+  Let's assume that the desired namespace is 'terrascan'.
+  ```bash
+  kubectl create namespace terrascan
+  ```
+
+### Deploying Terrascan Server
+
+Deploy terrascan in server mode operating in plain HTTP mode.
+
+1. Place your terrascan `config.toml` in the `base/config/` directory or edit the existing one.
+
+2. Deploy the terrascan server. Skip this step if you're aiming to setup terrascan in tls mode, terrascan webhook or
+   terrascan server for remote repository scan.
+
+   Note: Before running the command, please verify once that the `server/kustomization.yaml` is set with the desired parameters.
+
+    ```bash
+    kustomize build server/ | kubectl apply -f -
+    ```
+
+### Deploying Terrascan Server in TLS Mode
+
+Deploy terrascan in server mode operating in HTTPS mode.
+
+1. Follow Step 1 from `Deploying Terrascan Server` section
+
+2. Create a domain.cnf file.
+
+    ```bash
+    mkdir server-tls/certs
+    touch server-tls/certs/domain.cnf
+    cat << EOF > certs/domain.cnf
+    [req]
+    default_bits = 2048
+    prompt = no
+    default_md = sha256
+    x509_extensions = v3_req
+    distinguished_name = dn
+    [dn]
+    C = <My_Country>
+    ST = <My_State>
+    L = <My_Location>
+    O = <My_Organization>
+    emailAddress = <My_Email>
+    CN = terrascan.<TERRASCAN_NAMESPACE>.svc.cluster.local
+    [v3_req]
+    subjectAltName = @alt_names
+    [alt_names]
+    DNS.1 = terrascan.<TERRASCAN_NAMESPACE>.svc.cluster.local
+    >EOF
+    ```
+
+    **Note:** Please replace the placeholders like `<My_Country>`,`<My_State>` etc as per your requirements.
+
+3. Generate `server.key` and `server.crt`.
+
+    ```bash
+    openssl req -x509 -sha256 -nodes -newkey rsa:2048 -keyout server-tls/certs/server.key -out server-tls/certs/server.crt -config server-tls/certs/domain.cnf
+    ```
+
+4. Deploy the terrascan server. Skip this step if you're aiming to setup terrascan webhook or terrascan server for remote repository scan.
+
+   **Note:** Before running the command, please verify once that the `server-tls/kustomization.yaml` is set with the desired parameters.
+
+    ```bash
+    kustomize build server-tls/ | kubectl apply -f -
+    ```
+
+### Deploying Terrascan Server For Remote Repository Scan
+
+For scanning remote IaC file repositories, Terrascan must be provided with the required SSH keys to connect and clone the
+repository locally to scan it. The following steps will help in setting up for that.
+
+1. Follow steps 1-3 of the `Deploying TerraScan Server in TLS mode` section.
+
+2. Generate SSH keys and copy `~/.ssh/config`, `~/.ssh/known-hosts` and `~/.ssh/<generated_private_key>` to
+   `server-remote-repo-scan/.ssh/` directory. Replace `<SSH_KEY_NAME>` with your private ssh key's name in
```

Comment:
could not find the field <SSH_KEY_NAME> in the remote-repo-scan/kustomization.yaml file.